jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ByteArrayEndPoint.java

```diff
@@ -44,9 +45,9 @@ private static SocketAddress noSocketAddress()
         {
             return new InetSocketAddress(InetAddress.getByName("0.0.0.0"), 0);
         }
-        catch (Throwable x)
+        catch (IOException x)
```

Comment:
Can't change the `catch` clause, as it may throw other (runtime) exceptions that we want to catch.

Reply:
But those uncaught throwables are still thrown, they are just not wrapped in a needless RuntimeException any more.    I'll change to use the new `asRuntime` method to keep the catch the same, but still avoid the needless wraps.